manifests/init.pp

```diff
@@ -222,13 +222,13 @@
     false => true,
   }
 
-  if ($php::source and $php::template) {
+  if ($php::source != '' and $php::template != '') {
```

Comment:
All these checks should check both the presence of the variable and its emptyness: 
if ($php::source and $php::source != '' and $php::template and $php::template != '' )...